lti_emailer/canvas_api_client.py

```diff
@@ -21,6 +21,21 @@
 SDK_CONTEXT = SessionInactivityExpirationRC(**settings.CANVAS_SDK_SETTINGS)
 
 
+def get_section(canvas_course_id, section_id):
```

Comment:
Do we need to cache these again individually? Couldn't we just grab them from get_sections below?